catalog-info.yaml

```diff
@@ -36,6 +39,46 @@ spec:
         SLACK_NOTIFICATIONS_ALL_BRANCHES: "false"
         SLACK_NOTIFICATIONS_ON_SUCCESS: "false"
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-docs-pr
+  description: Build the docs pull requests
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/docs-build-pr
+
+spec:
+  type: buildkite-pipeline
+  owner: group:docs
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: docs / build-pr
+    spec:
+      repository: elastic/docs
+      pipeline_file: ".buildkite/build_pr_pipeline.yml"
+      provider_settings:
+        trigger_mode: none # triggered via webhook and the buildkite-pr-bot
+        skip_intermediate_builds: false
```

Comment:
I'm getting a YAML schema error locally on this line. I think it needs to be up one level (a sibling of `repository`).